packages/js-moi-providers/__tests__/jsonrpc-provider.test.ts

```diff
@@ -123,6 +126,23 @@ describe("Test JsonRpcProvider Query Calls", () => {
       });
     });
 
+    describe("getInteractionByTesseract", () => {
+      it('should return the interaction by tesseract', async () => {
+        const interactions = await provider.getInteractionByTesseract(address, undefined, "0x0");
+
+        expect(interactions).toBeDefined();
+        expect(interactions.hash).toBe(ixReceipt.ix_hash);
+      });
+
+      it("should return the interaction by tesseract without address", async () => {
+        const tesseract = await provider.getTesseract(address, true);
+        const interaction = await provider.getInteractionByTesseract({ tesseract_hash: tesseract.hash }, "0x0");
+
+        expect(interaction).toBeDefined();
+        expect(interaction.ts_hash).toBe(tesseract.hash);
```

Comment:
replace this with not null, and repeat the same for all the other occurrences.

Reply:
ff1526080cac6383d6521a80b129d8a610dc7f99